app/records-interfaces/allocation-records-interface.coffee

```diff
@@ -0,0 +1,13 @@
+null 
+
+### @ngInject ###
+global.cobudgetApp.factory 'AllocationRecordsInterface', (config, BaseRecordsInterface, AllocationModel) -> 
+  class AllocationRecordsInterface extends BaseRecordsInterface
+    model: AllocationModel
+    constructor: (recordStore) ->
+      @baseConstructor recordStore
+      @restfulClient.apiPrefix = config.apiPrefix 
```

Comment:
nevermind, [as every `restfulClient` is a new instance](https://github.com/loomio/angular_record_store/blob/master/base_records_interface.coffee#L13).